internal/cli/cmd/dac/diff/diff.go

```diff
@@ -82,34 +83,36 @@ func (o *option) Validate() error {
 }
 
 func (o *option) Execute() error {
+	// Create the output folder (+ any parent folder if applicable) where to store the diff files
+	err := os.MkdirAll(config.Global.Dac.OutputFolder, os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("error creating the output folder: %v", err)
+	}
+
+	var execResult strings.Builder
 	for _, preview := range o.dashboards {
 		project := resource.GetProject(preview.GetMetadata(), o.Project)
 		dashboard, err := o.apiClient.V1().Dashboard(project).Get(preview.Metadata.Name)
 		if err != nil {
-			return err
+			execResult.WriteString(fmt.Sprintf("No dashboard %s found in project %s, skipping diff generation: %v\n", preview.Metadata.Name, project, err))
```

Comment:
you could just write the message directly into the output. Would be simpler don't you think ? And also it will show the progress of the command

Reply:
For example, in case of error, you could use the logger in this case instead of using the output handler. Logger will write the log on the error output